tests/unit/test_yaml_utils.py

```diff
@@ -0,0 +1,81 @@
+# (c) 2012-2019, Ansible by Red Hat
+#
+# This file is part of Ansible Galaxy
+#
+# Ansible Galaxy is free software: you can redistribute it and/or modify
+# it under the terms of the Apache License as published by
+# the Apache Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# Ansible Galaxy is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# Apache License for more details.
+#
+# You should have received a copy of the Apache License
+# along with Galaxy.  If not, see <http://www.apache.org/licenses/>.
+
+import os
+
+import pytest
+
+from galaxy_importer.utils import yaml as yaml_utils
+
+
+@pytest.fixture
+def tmp_file():
+    try:
+        dir = os.path.dirname(os.path.dirname(os.path.dirname(__file__)))
+        tmp_file = os.path.join(dir, 'env.yml')
+        yield tmp_file
+    finally:
+        os.remove(tmp_file)
+
+
+EX_ENV_YAML = """---
+version: 1
+dependencies:
+  galaxy: requirements.yml
+  python: requirements.txt
+  system: bindep.txt
+
+additional_build_steps:
+  prepend: |
+    RUN whoami
+    RUN cat /etc/os-release
+  append:
+    - RUN echo This is a post-install command!
+    - RUN ls -la /etc
```

Comment:
the additional_build_steps wouldn't be supported inside of collections. Collections could do all kinds of things which would be incompatible with other collections, so that's only left as something the user can add with writing their execution environment definition. It shouldn't need to be processed by the galaxy importer at all.

Reply:
Thanks for the clarification, I'll update my PR accordingly.